zetta_utils/bcube/bcube.py

```diff
@@ -83,7 +101,7 @@ def from_coords(
         cls,
         start_coord: VecT,
         end_coord: VecT,
-        resolution: VecT,
+        resolution: Optional[VecT] = None,
```

Comment:
It almost does the same thing as setting the default to be (1, 1, 1), which is probably cleaner. The reason why it's set to `None` is because VecT is not known - it can be a Sequence or a Vec3D. I can rebase to incorporate Nico's change and then set the default to `Vec3D(1,1,1)`

Reply:
Oh, even with Nico's changes this doesn't work because VecT is not known at runtime. Not sure what's a good way to do this. Probably another "apply alchemy to BoundingCube and the problem will go away" item.

Reply:
How about we keep Bcube as is to merge this in ASAP, but then put on a high priority issue to make a 3D-only bbox?

Reply:
Sounds good.